packit_service/worker/helpers/testing_farm.py

```diff
@@ -410,11 +410,11 @@ def get_latest_copr_build(
             owner=self.job_owner,
             target=target,
         )
-        if not copr_builds:
+        try:
+            return next(iter(copr_builds))
+        except StopIteration:
             return None
```

Comment:
_Python ships whole universe with the interpreter, yet it doesn't have `first_or_none`_ /o\

Reply:
Actually, `.first()` does that, i.e. it returns `None` in case of empty results, but mypy screams because
`error: "Iterable[CoprBuildTargetModel]" has no attribute "first"`
So I did it this way to make it happy because I was lazy to add `CoprBuildTargetModel.get_first_by()`.

Reply:
I have a feeling the `.first()` comes from SQLAlchemy and is not supported for any iterable /o\

Reply:
Oh, you didn't mean the SQLAlchemy's first() but something which would do that `try next(iter())) except StopIteration` stuff. I see.
That, or at least [has_next()](https://stackoverflow.com/questions/1966591/has-next-in-python-iterators) would be nice.